spring-cloud-starter-stream-processor-tasklaunchrequest-transform/pom.xml

```diff
@@ -18,6 +18,7 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-task-stream</artifactId>
+			<version>2.0.0.BUILD-SNAPSHOT</version>
```

Comment:
We don't need to set this here.

The change in the `tasklaunchrequest-transform-app-dependencies/pom.xml` is fully enough.

Reply:
good catch!